stories/dialogModal/index.stories.tsx

```diff
@@ -99,21 +179,21 @@ export default {
           <ArgsTable />
           <Heading>Stories</Heading>
           <Description>
-            Source code for the stories can be found in the individual story
-            canvas tabs. Keeping the modals closed-by-default on the docs page
-            (unlike the open-by-default configuration on the canvas pages)
-            interferes with Storybook source inference.
+            The below stories are the not-open-by-default versions of the
```

Comment:
Hopefully the description here (and in ImmersiveModal) adequately explains the duplication. The stories do not accept props easily, and so when trying to integrate the stories manually in the Docs page there were either: (1) Auto-open issues, or (2) The usage of the component directly outside of the Story broke styling. I decided this was the easiest way to work around the issue.